plugin/al/translators/tests/CMakeLists.txt

```diff
@@ -1,5 +1,53 @@
+set(TEST_NAME TestAdditionalTranslators)
+
+set(path 
+    "${CMAKE_INSTALL_PREFIX}/lib"
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib"
+    "$ENV{PATH}"
+)
+
+set(pythonPath 
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib/python"
+    "$ENV{PYTHONPATH}"
+)
+
+set(mayaPluginPath 
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/plugin"
+)
+
+if(IS_WINDOWS)
+    string(REPLACE ";" "\;" pythonPath "${pythonPath}")
```

Comment:
turn into a utility function - see note for plugin/al/mayautils/AL/maya/tests/mayaplugintest/CMakeLists.txt